experimental/packages/otlp-exporter-base/test/browser/fetch-transport.test.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as sinon from 'sinon';
+import * as assert from 'assert';
+import { createFetchTransport } from '../../src/transport/fetch-transport';
+import {
+  ExportResponseRetryable,
+  ExportResponseFailure,
+  ExportResponseSuccess,
+} from '../../src';
+
+const testTransportParameters = {
+  url: 'http://example.test',
+  headers: () => ({
+    foo: 'foo-value',
+    bar: 'bar-value',
+    'Content-Type': 'application/json',
+  }),
+};
+
+const requestTimeout = 1000;
+const testPayload = Uint8Array.from([1, 2, 3]);
+
+describe('FetchTransport', function () {
+  afterEach(() => {
```

Comment:
nit: for consistency and accessing mocha context via `this`.
```suggestion
  afterEach(function() {
```